README.md

```diff
@@ -1,4 +1,16 @@
 [![Python package](https://github.com/mobiusml/aana_sdk/actions/workflows/python-package.yml/badge.svg)](https://github.com/mobiusml/aana_sdk/actions/workflows/python-package.yml)
+![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)
```

Comment:
When clicked it doesn't show the license.

Reply:
Done